### PR TITLE
Allow --config in kubernetes_bazel scenario

### PR DESCRIPTION
**Pull request by fejta**
Allow jobs to pass config options like `--config=ci` into the scenario

/assign @BenTheElder @krzyzacy @spiffxp

### CODE REVIEW

File: scenarios/kubernetes_bazel.py [hunk 1]
```diff
@@ -39,21 +39,28 @@ def check_output(*cmd):
...
 
-    def check(self, *cmd):
-        """wrapper for check('bazel', *cmd) that respects batch"""
+    def _commands(self, cmd, *args, **kw):
+        commands = ['bazel']
         if self.batch:
-            check('bazel', '--batch', *cmd)
-        else:
-            check('bazel', *cmd)
+            commands.append('--batch')
```
Comment (BenTheElder) [inline scenarios/kubernetes_bazel.py]: --batch has been deprecated for a while (~one year, 0.15.0), we could probably get rid of this.

https://github.com/bazelbuild/bazel/commit/2d53deee92ed6134e861ab6d0a8a0355510bb57d

**Comment by BenTheElder** [comment]
pylint disagrees with your indentation

File: scenarios/kubernetes_bazel.py (hunk 1 above)
Comment (fejta) [reply scenarios/kubernetes_bazel.py]: Done

**Comment by fejta** [comment]
2 spaces forever

**Pull request merged**